pkg/runtime/optimize/dml/select.go

```diff
@@ -87,6 +87,24 @@ func optimizeSelect(ctx context.Context, o *optimize.Optimizer) (proto.Plan, err
 
 	log.Debugf("compute shards: result=%s, isFullScan=%v", shards, fullScan)
 
+	//use route hint when fullScan
+	hints := rcontext.RouteHints(ctx)
```

Comment:
It can be based https://github.com/arana-db/arana/pull/315 on implement route

Reply:
fine, i will fix after #315 merged